DatabaseHandler/MS2DBModel.edmx.sql

```diff
@@ -0,0 +1,100 @@
+
+-- --------------------------------------------------
+-- Entity Designer DDL Script for SQL Server 2005, 2008, 2012 and Azure
+-- --------------------------------------------------
+-- Date Created: 01/03/2021 19:59:22
+-- Generated from EDMX file: D:\Maplestory 2 Emulator\MapleStory 2 Emu\DatabaseHandler\MS2DBModel.edmx
```

Comment:
This seems auto-generated, if so, does this need to be in the repo?

Reply:
This is what you need to generate your database. its generated from the entity framework.